jobs/async-upload/README.md

```diff
@@ -0,0 +1,43 @@
+# Kubeflow Model Registry – Async Upload Job
+
+> Lightweight, non‑root Python Job containerised for **transferring model artefacts** between storage back‑ends (S3, OCI, PVC, …) and registering them in **Kubeflow Model Registry**. Born out of the discussion in [kubeflow/model-registry #1108](https://github.com/kubeflow/model-registry/issues/1108).
+
+---
+
+## Quick start
+
+```bash
+# 1 – Build & tag ( reproducible build args are optional )
+docker build \
+  --build-arg VCS_REF=$(git rev-parse --short HEAD) \
+  --build-arg BUILD_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
+  -t quay.io/<org>/async-upload-job:$(git rev-parse --short HEAD) .
+
+# 2 – Push to your registry
+docker push quay.io/<org>/async-upload-job:<tag>
+
+# TODO: Run locally...
+```
+
+---
+
+## Security & supply‑chain
+
+- **Non‑root runtime**
+- **Distroless‑ish** slim base, no build tools in final layer
+- **SBOM generation** – use `syft packages dir:.` in CI
+- **Image signing** – add `cosign sign --key cosign.key <image>`
+
+---
+
+## License
+
+Licensed under the **Apache License 2.0** – see [`LICENSE`](../../LICENSE) in root for details.
+
+---
```

Comment:
I don't think this part is needed, because we have already coverage from the main repo (I would avoid duplication), further I don't think we're cosigning container images ootb